index.html

```diff
@@ -47,7 +47,7 @@ <h1 style="padding-left: 60px">Open Models == Better Policy</h1>
                                 News
                             </div>
                             <div class="card-body">
-                                <p>Read the latest updates from PSL.</p>
+                                <p>Our aim is to keep you up to date with key developments from library projects. Review the news here or sign up to our newsletter to stay up to date.</p>
```

Comment:
@MattHJensen I think this would read better if it were `or sign up to receive our newsletter`. Other than that looks good to me.

Reply:
Thanks @andersonfrailey, fixed in the latest commit. Merging. Feel free to make additional changes to the text as you see fit through new commits.